packages/api-client/src/requestManager.ts

```diff
@@ -93,7 +96,18 @@ export class RequestManager<API extends API.Spec> {
         >;
       } else {
         const json = await response.json();
-        // TODO validate
+
+        const validationResult = this.validator.validateResponse(
+          { path, method, ...requestData },
+          json,
+        );
+        if (validationResult !== true) {
+          console.warn(
+            `Validation Error: (${method}) ${path} did not match the expected value for this route`,
+          );
+          console.warn(validationResult);
+        }
```

Comment:
Good question! One easy way: run `yarn dev` in `packages/web-component` then navigate to `http://localhost:5000` to find a web page with some test embedded prompts.